tests/FSharpLint.Core.Tests/Rules/Conventions/FavourStaticEmptyFields.fs

```diff
@@ -118,6 +118,39 @@ type Person =
         Assert.IsTrue this.ErrorsExist
         Assert.IsTrue (this.ErrorMsg.Contains "String.Empty")
 
+    [<Test>]
+    member this.FavourStaticEmptyFieldsShouldProduceError13() =
+        this.Parse """
+let foo =
+    {
+        Range = Range.mkRange "" pos pos
+        Message = Resources.GetString("RulesTypographyTrailingLineError")
+        SuggestedFix = None
+        TypeChecks = List.Empty
```

Comment:
@webwarrior-ws let's remove unnecessary record elements to simplify the testcase